### ISSUE TITLE
[Balance] Base submarine gun power consumption is too low

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#9620]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As a baseline, we'll use a low-energy supercapacitor at 50% charge rate (250kW), which roughly charges at 2% of its capacity per second

**Pulse laser** only uses 2% of a supercapacitor charge per shot, at 1 shot per second
A supercapacitor at 50% charge rate will recharge faster than a laser that is continuously shooting
Full box: 45 shots
![image](https://user-images.githubusercontent.com/73229309/180267951-3aed7b07-ae84-4c2d-883e-5ed6c606a702.png)

**Chaingun** power consumption matches charge rate at around 70% charge rate (500kW)

**Railguns and Coilguns** are a little more balanced, it takes a charge rate of about 80% (600+kW) to not run out of power

You'd expect charge rate to be cranked up to 100% to perfectly match power consumption, effectively giving you "infinite" energy at the cost of overheating, but all guns match it way before talents and upgrades are applied

**Assuming:**
- most players/subs do use one supercap per gun
- most guns won't be firing constantly
- not all guns firing at the same time
- not factoring EDCs

You can make a case for nerfing the guns' power consumption in general. 
It's more intuitive to increase base power consumption for all and let sub builders tweak supercap stats if shared across multiple guns (example: Barsuk, Azimuth)

In short: 
-Increase laser gun power consumption by 5x (really)
-Increase power consumption of all other guns slightly

**Comment by JeroonBosch** [comment]
Pulse Laser power consumption didn't take into account the charge time (so it overestimated the fire rate). 5x the power consumption makes it equal to the other guns. Railgun was also using less power than maybe intended so these have been equalized

**Comment by Tsunder** [comment]
I believe this is related to "new supercap defaults/ used values means supercap never run dry" #8502

**Comment by ubertpendragon** [comment]
Pulse laser power consumption is still the same on unstable as I write this 🤔

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/acba217e8ed3365dc77e26e5e98f7f4fc0a29971. Note for testing: I committed this on the subgrades branch to avoid merge conflicts

**Comment by Rokvach** [comment]
Tested, no issues. Closing.
